inventory/helper.ini

```diff
@@ -1,2 +1,3 @@
 [all]
 helper ansible_connection=local
+kvm ansible_host=KVM_HOSTNAME
```

Comment:
I am not sure if I understand why the kvm host is introduced here. I think if somebody wants to use a remote host, they could set ansible_host=x.x.x.x ansible_connection=ssh ? Would that not work? Or perhaps I don't understand the purpose here.

Reply:
The reasons are here https://github.com/noseka1/openshift-auto-upi/pull/22#discussion_r493193736.